projects/ghostscript/project.yaml

```diff
@@ -1,6 +1,7 @@
 homepage: "https://ghostscript.com"
 primary_contact: "skau@google.com"
 auto_ccs:
+  - "henry.stiles@artifex.com"
```

Comment:
I don't think you can access oss-fuzz's web UI without a google/gmail account. Does the maintainer have one? Otherwise they will only be able to see summaries of the bugs in the issue tracker, but won't be able to access testcases or the full stack trace.

Reply:
`dig MX artifex.com` seems to show that it's a gsuite

Reply:
Thanks Alex!